--- tmp/pipe/linux/CMakeLists.txt.orig  2026-05-25 23:56:56.924680270 +1000
+++ tmp/pipe/linux/CMakeLists.txt       2026-05-25 23:57:12.001012416 +1000
@@ -196,9 +196,7 @@
     #       setting GIT_DIR=. which fucks this up. So we have to write our own
     #       DOWNLOAD_COMMAND which explicitly unsets GIT_DIR to solve that.
     #GIT_REPOSITORY https://github.com/rolandoislas/drc-hostap.git
-    DOWNLOAD_COMMAND
-        ${CMAKE_COMMAND} -E env GIT_DIR=
-        ${GIT_EXECUTABLE} clone --branch master --depth 1 https://github.com/rolandoislas/drc-hostap.git "${HOSTAP_DIR}"
+    DOWNLOAD_COMMAND ${CMAKE_COMMAND} -E copy_directory @DRC_HOSTAP_SRC@ "${HOSTAP_DIR}"
     CONFIGURE_COMMAND ${CMAKE_COMMAND} -E copy_if_different conf/wpa_supplicant.config wpa_supplicant/.config
     BUILD_COMMAND env ${HOSTAP_ENV} make -C wpa_supplicant -j${CMAKE_BUILD_PARALLEL_LEVEL} wpa_supplicant libwpa_client.a
     INSTALL_COMMAND ""
